CMakeLists.txt

```diff
@@ -35,13 +36,22 @@ mark_as_advanced(MATERIALX_PYTHON_EXECUTABLE)
 mark_as_advanced(MATERIALX_PYTHON_OCIO_DIR)
 mark_as_advanced(MATERIALX_PYTHON_PYBIND11_DIR)
 
+# Shader generation options
+option(MATERIALX_BUILD_GEN_OSL "Build the OSL shader generator back-end." ON)
```

Comment:
For discussion:  Since the new MaterialXGenGlsl and MaterialXGenOsl modules are lightweight and introduce no external library dependencies, perhaps it would be better to remove these two options, and to build these new modules in all configurations.  Developers can still choose which modules they want to link into their applications, so they're not compelled to include any more code than they need.

Reply:
We can always build these as long as it's agreeable that the Python package always includes these 2 sub-modules.

Reply:
We'll change the build rules so both are built by default with no options.